.github/workflows/pkgci_regression_test.yml

```diff
@@ -313,3 +327,26 @@ jobs:
             --log-cli-level=info \
             --retries 7
           echo "$(<job_summary.md )" >> $GITHUB_STEP_SUMMARY
+          rm job_summary.md
+
+      # TODO(scotttodd): update these golden values, retry count, etc.
+      - name: "Running SDXL rocm pipeline benchmark (mi300)"
+        if: contains(matrix.name, 'rocm_mi300_gfx942')
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest SHARK-TestSuite/iree_tests/benchmarks/sdxl/benchmark_sdxl_rocm.py \
+            --goldentime-rocm-e2e-ms 1336.0 \
+            --goldentime-rocm-unet-ms 340.0 \
+            --goldentime-rocm-clip-ms 17.0 \
+            --goldentime-rocm-vae-ms 291.0 \
+            --goldendispatch-rocm-unet 1714 \
+            --goldendispatch-rocm-clip 1569 \
+            --goldendispatch-rocm-vae 248 \
+            --goldensize-rocm-unet-bytes 2062938 \
+            --goldensize-rocm-clip-bytes 780328 \
+            --goldensize-rocm-vae-bytes 757933 \
+            --gpu-number 0 \
```

Comment:
Did you want to run with a specific GPU here?

I imagine we could do something when adding the runner (Docker?) to
* Have multiple runners on the same physical machine
* Only make a single GPU visible per runner
* Let each job use that one GPU by just using index 0 (or leaving it up to automatic selection instead of being explicit)

Reply:
Yeah have to research more into how all that works with the permissions and everything, but would be cool to have multiple runners off the same machine